addons/io_hubs_addon/debugger.py

```diff
@@ -59,15 +59,61 @@ def poll(cls, context: Context):
 
     def execute(self, context):
         try:
+            selected_obs = bpy.context.selected_objects
+            active_ob = bpy.context.active_object
+
+            viewpoint = None
+            if context.scene.hubs_scene_debugger_room_export_prefs.avatar_to_viewport:
+                area = find_area()
+                if area is not None:
+                    r3d = area.spaces[0].region_3d
+                    view_mat = r3d.view_matrix.inverted()
+                    loc, rot, _ = view_mat.decompose()
+                    from mathutils import Matrix, Vector, Euler
+                    from math import radians
```

Comment:
Locally imported modules aren't bad per se, but (in my opinion) keeping imports at the top makes it easier to see what a file depends on, gives a consistent spot to look for this information, and prevents duplication.  Sometimes they have to be imported in the middle of code to resolve dependency issues or other bugs and that's what I thought was usually the reason they had been added throughout the code (and that's the main reason I asked, because I wondered if you were working around a bug).  In certain cases they may cause a slowdown in the middle of execution instead of at the beginning (but that's likely not an issue here).  Also, PEP8 does specify for imports to be placed at the top: https://peps.python.org/pep-0008/#imports
This page also gives some reasons for placing them at the top: https://www.quora.com/Should-import-be-above-all-from-statements-in-Python-If-yes-why